### ISSUE TITLE
[HW] Lokkeshwaran Jayabalan

### ISSUE THREAD

**Issue by lokkeshjaya** [datastaxdevs/workshop-k8ssandra#14]
**Name:** Lokkeshwaran J
**Linkedin Profile:** https://www.linkedin.com/in/lokkeshwaran/

Attach the homework screenshots below:
-----------------------------------------

![4](https://user-images.githubusercontent.com/39264572/111266088-0c7daa80-8650-11eb-9d71-210e499c1d8e.png)
![5](https://user-images.githubusercontent.com/39264572/111266090-0e476e00-8650-11eb-8744-717be9aea6dd.png)
![6](https://user-images.githubusercontent.com/39264572/111266094-0f789b00-8650-11eb-88f2-9d5238b2ddcc.png)
![8](https://user-images.githubusercontent.com/39264572/111266097-10113180-8650-11eb-99ee-519e0ccbbe39.png)
![1](https://user-images.githubusercontent.com/39264572/111266100-10113180-8650-11eb-9d3f-3a5e4ce32a74.png)
![2](https://user-images.githubusercontent.com/39264572/111266104-11425e80-8650-11eb-9f2e-6e9d6886cd21.png)
![3](https://user-images.githubusercontent.com/39264572/111266107-11daf500-8650-11eb-875c-c229bfb989d0.png)

**Comment by lokkeshjaya** [comment]
Interesting Workshop to get hands dirty on K8 + Cassandra 👍  Thanks Datastax Team @HadesArchitect

**Comment by HadesArchitect** [comment]
Great job @lokkeshjaya! Thank you so much! Accepted with no doubts!

**Comment by HadesArchitect** [comment]
Are you from Chennai? I see the city in the screenshot of petclinic database. Such a coincidence, a great friend of mine is from Chennai! 😁

**Comment by lokkeshjaya** [comment]
Yes, I'm from Chennai. Now you have another Friend from Chennai ;)

**Comment by HadesArchitect** [comment]
Here we go! Sorry for the slow down, I was on vacation 😓 

https://badgr.com/public/assertions/r0_7521fTbirUxyqcrcLfg?identity__url=https:%2F%2Fwww.linkedin.com%2Fin%2Flokkeshwaran%2F

**Comment by lokkeshjaya** [comment]
Hi @HadesArchitect  .. How do i get a link to share it in LinkedIN ? Any help on that would be great.

**Comment by lokkeshjaya** [comment]
No worries. Figured out a way to post it on LinkedIn.  Thanks for the Badge 👍
